### DISCUSSION TITLE
Default theme not rendering properly

### DISCUSSION THREAD

**Discussion by arung2207** [serenity-is/Serenity#6403] [Help] [answered]
After creating a new project using the Serene template in VS2022 I have migrated to .Net 6.0 with the steps described at https://github.com/serenity-is/Serenity/issues/6337. The default theme is not appearing well, please see the screenshot below. I have tried it on Edge, Chrome and FireFox also with similar results. Am I missing a configuration? Is any of the StartSharp themes available for evaluation?

![image](https://user-images.githubusercontent.com/3456937/170239711-98275590-22bc-4a27-9aae-bb9c2a625af8.png)

**Comment by wezmag** [comment, marked as answer]
Looks like some CSS files are not generated.
Try update your `sergen` to `6.0.1`
``` cli
dotnet tool update sergen
```
then restore
``` cli
dotnet sergen restore
```
If there are any errors when running `restore`, please share them.

**Reply by arung2207** [reply]
Restore worked, thanks!
